pkg/virt-launcher/virtwrap/network/binding_test.go

```diff
@@ -0,0 +1,166 @@
+package network
+
+import (
+	"fmt"
+	"net"
+	"runtime"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/golang/mock/gomock"
+	"github.com/vishvananda/netlink"
+
+	kubevirtv1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/kubevirt/pkg/network/cache"
+	"kubevirt.io/kubevirt/pkg/network/cache/fake"
+	netdriver "kubevirt.io/kubevirt/pkg/network/driver"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+)
+
+var _ = Describe("BindingMechanism", func() {
+	var (
+		TestRunPlug = func(driver BindMechanism, podInterfaceName string) {
```

Comment:
You've resolved the comment, but I still see the functions defined in var blocks.

Reply:
I think you are reviewing all code TestRunPlug is not in a var block anymore.

Reply:
Hm, no. I'm reviewing commit https://github.com/kubevirt/kubevirt/pull/5747/commits/fbea3ded9b59f43bdec9564214b1979dbc6f2094 (the one before last).

You update this in the next commit, which, imo, is not the correct one.

Reply:
Done